install/tests/Makefile

```diff
@@ -101,11 +137,8 @@ cleanup:
 
 get-results:
 	@echo "If you have gotten this far, it means your setup succeeded"
-	@echo
-	@echo "URL of your setup is: "https://$(TF_VAR_TEST_ID).gitpod-self-hosted.com""
-	@echo
+	@echo "The IP address of you setup is "https://$(TF_VAR_TEST_ID).gitpod-self-hosted.com""
```

Comment:
Note: will this subdomain need to change given the issues we've had with domain level wildcard letsencrypt issuance?

Reply:
We will highly likely stick with the same subdomain, just we will try and re-use the certs issued across multiple sub-domains(max 200 subdomains can be added under a certificate, iirc). In reality, I don't think we will hit a limit even if we create new certs per domain. This week it so happened because I was running too many tests on my own